assets/js/theme/common/product-details.js

```diff
@@ -7,6 +7,8 @@ import _ from 'lodash';
 import Wishlist from '../wishlist';
 import { normalizeFormData } from './utils/api';
 
+require('formdata-polyfill');
```

Comment:
Could we move this import to polyfills.js file?

Reply:
changed